builtin/builtin.mbti

```diff
@@ -360,7 +360,8 @@ impl Int {
   to_json(Int) -> Json
   to_string(Int) -> String
   to_uint(Int) -> UInt
-  until(Int, Int, ~step : Int = ..) -> Iter[Int]
+  until(Int, Int, ~step : Int = .., ~inclusive : Bool = ..) -> Iter[Int]
```

Comment:
This is too repetitive though, they serve the same purpose (default `step` for `until` is 1, behave the same as `upto`). I think we can keep only `until` and desugar to it.

Reply:
The compiler should work just fine BTW